CHANGELOG/RELEASE-NOTES.md

```diff
@@ -1,5 +1,30 @@
 # k8ssandra-operator - Release Notes
 
+## v1.5.0
+
+
+### New Metrics Endpoint
+
+As of v1.5.0, we are introducing a new metrics endpoint which will replace MCAC in an upcoming release and is built directly in the Management API.  
```

Comment:
Do users know what MCAC is..?

Reply:
Well, we've always mentioned it as part of the main components in K8ssandra: https://docs.k8ssandra.io/
Maybe I should add a link to the github project?